docker-compose.yml

```diff
@@ -14,5 +14,6 @@ services:
       - HOST_UID=${HOST_UID:-1000}
       - HOST_GID=${HOST_GID:-1000}
       - TZ=${TZ:-America/Sao_Paulo}
-      - XDEBUG_CONFIG=${XDEBUG_CONFIG:-"client_host=172.17.0.1 client_port=9003 start_with_request=yes"}
+      - XDEBUG_CONFIG=${XDEBUG_CONFIG:-client_host=172.17.0.1 client_port=9003 start_with_request=yes}
```

Comment:
Esse valor fixo das variáveis `client_host` e `client_port` estão corretos ?

Reply:
O host é o IP default do gateway (host) no Docker
A porta é a padrão do xdebug.
Caso alguém por algum motivo pegue um IP que não seja o padrão do Docker, pode criar um arquivo `.env` e sobrescrever a environment `XDEBUG_CONFIG`

Reply:
So would be great if we have a `.env` and  have variables appointing for it

Reply:
I think not because have default values.
The .env is only necessary if need to use a different value of default.
By example we can create a .env file only with `HOST_UID=1001`, the default is 1000.

Reply:
Thinking on this way we also can remove `client_port` since the default value is `9003`

https://xdebug.org/docs/all_settings#client_port

Reply:
I agree.